### PR TITLE
integrate vincia

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

**Comment by mkirsano** [comment]
This is a plugin for pythia8. The code that uses it in CMSSW Pythia8Interface will follow (there is a prototype).

File: vincia-toolfile.spec [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <lib name="vincia"/>
+  <lib name="mg4helas"/>
+  <client>
+    <environment name="VINCIA_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR" default="$VINCIA_BASE/lib"/>
+    <environment name="INCLUDE" default="$VINCIA_BASE/include"/>
+  </client>
+  <runtime name="VINCIADATA" value="$VINCIA_BASE/share/Vincia/xmldoc"/>
+  <use name="root_cxxdefaults"/>
+  <use name="cxxcompiler"/>
```
Comment (smuzaffar) [inline vincia-toolfile.spec]: please drop cxxcompiler dependency.

File: vincia.spec [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### RPM external vincia 1.2.02
+
+Requires: pythia8
+
+Source: https://www.hepforge.org/archive/vincia/%{n}-%{realversion}.tgz
+
+%define keep_archives true
+
+%if "%{?cms_cxxflags:set}" != "set"
+%define cms_cxxflags -std=c++0x
```
Comment (smuzaffar) [inline vincia.spec]: no need to set this, not used any where in this spec.

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Comment by davidlange6** [comment]
hi @mkirsano - indeed, this PR should be made instead for 9_3_X branch. Sorry for the trouble

**Comment by smuzaffar** [comment]
closing it in favour of 93X PR #3150 . Please open an issue if this is needed in older releases

**Pull request closed without merging**